modules/issue_tracker/php/endpoints/attachment.class.inc

```diff
@@ -0,0 +1,245 @@
+<?php declare(strict_types=1);
+
+namespace LORIS\issue_tracker\endpoints;
+
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \LORIS\Middleware\ETagCalculator;
+use \LORIS\api\Endpoint;
+
+/**
+ * The Attachment class for requests from issue_trackers.
+ *
+ * @category Loris
+ * @package  IssueTracker
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Attachment extends Endpoint implements ETagCalculator
+{
+    public $skipTemplate = true;
```

Comment:
$skipTemplate is a public variable of NDB_Page
Attachment do not extend NDB_Page